### ISSUE TITLE
spark-shell

### ISSUE THREAD

**Issue by thbeh** [gustavonalle/spark-docker#1]
Hi, I was searching around for docker image on spark and came across your code. While I managed to get bot master and worker to run but I have some issue with spark-shell. Not sure whether it is OS (notice that you are using alphine) related but it complain of of not able to find  -

java.lang.IllegalArgumentException: java.lang.UnsatisfiedLinkError: /tmp/snappy-1.1.2-8d391784-c1c5-4f60-a1cf-d5066bf5b5f7-libsnappyjava.so: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /tmp/snappy-1.1.2-8d391784-c1c5-4f60-a1cf-d5066bf5b5f7-libsnappyjava.so)

**Comment by gustavonalle** [comment]
Yes. snappy is not working at the moment in the image. You can use lz4 though, for example
```
docker exec -it spark-master /usr/local/spark/bin/spark-shell --master spark://172.17.0.3:7077 --conf spark.io.compression.codec=lz4 
```
